lapis/cmd/nginx.moon

```diff
@@ -94,11 +97,12 @@ class NginxRunner
         return @_nginx_path
 
     for prefix in *@nginx_search_paths
```

Comment:
can we swap how these loops are nested so that the existing search order is preserved with this change

Reply:
Sure yes, all the `nginx` then all the `openresty`